libdnf/module/module_sack_impl.hpp

```diff
@@ -34,6 +34,10 @@ class ModuleSack::Impl {
     Impl(const BaseWeakPtr & base) : base(base), pool(pool_create()) {}
     ~Impl() { pool_free(pool); }
 
+    const std::vector<std::unique_ptr<ModuleItem>> & get_modules();
+
+    void module_filterring();
```

Comment:
There should be only one "r" in the word "filtering".

Reply:
Thank you

Reply:
Fixed